applications/CoSimulationApplication/python_scripts/base_classes/co_simulation_data_transfer_operator.py

```diff
@@ -11,14 +11,32 @@ def __init__(self, settings):
         self.settings = settings
         self.settings.ValidateAndAssignDefaults(self._GetDefaultSettings())
         self.echo_level = self.settings["echo_level"].GetInt()
+        self.__checked_combinations = []
 
     def TransferData(self, from_solver_data, to_solver_data, transfer_options):
-        raise NotImplementedError("This function has to be implemented in the derived class!")
+        # 1. Check if specified transfer options are available
+        self._CheckAvailabilityTransferOptions(transfer_options)
```

Comment:
This function _CheckAvailability is to be implemented by the derived classes or ?

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/64a93514f881b2cf1351baedba03306d0fab1134/applications/CoSimulationApplication/python_scripts/base_classes/co_simulation_data_transfer_operator.py#L28-L30